index.js

```diff
@@ -50,4 +44,48 @@ fetch(
   })
   .then(albums => {
       console.log(albums)
+      albums.forEach(album => {
+        createAlbumContainer(album)
+      })
+
   })
+
+function createAlbumContainer(album) {
+
+  // <div class="album">
+  //   <img src="albumimgurl.com" alt="">
+  //   <h2>Name of the artist</h2>
+  //   <h2>Name of the album</h2>
+  //   <ol class="tracklist">
+  //       <li class="track"></li>
+  //       <li class="track"></li>
+  //       <li class="track"></li>
+  //       <li class="track"></li>
+  //   </ol>
+  // </div>
+
+  console.log(album.href)
+  let albumContainer = document.createElement('div')
+  
+  let albumImg = new Image()
+  albumImg.src = album.images[0].url
+
+  let artistNameHeader = document.createElement('h2')
+  let albumNameHeader = document.createElement('h2')
+  let tracklist = document.createElement('ol')
+  
+  album.tracks.items.forEach(track => {
+    let trackLi = document.createElement('li') 
+    trackLi.innerHTML = track.name
+    tracklist.appendChild(trackLi)
+  })
+
+  albumContainer.appendChild(albumImg)
+  albumContainer.appendChild(artistNameHeader)
+  albumContainer.appendChild(albumNameHeader)
+  albumContainer.appendChild(tracklist)
+
+  document.querySelector("#albums").appendChild(albumContainer)
+}
+
+const $ = document.querySelector
```

Comment:
why is there an unused variable here? Do we need it? Otherwise get rid of it.

Reply:
oh shit my bad, ill get rid of that